targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/TARGET_MTB_STM_L475/system_clock.c

```diff
@@ -200,6 +201,7 @@ uint8_t SetSysClock_PLL_HSE(uint8_t bypass)
         return 0; // FAIL
     }
 
+#if 0
```

Comment:
What's the reason for leaving this block of code in if it's never going to be run?

Reply:
No reason. I'll take it out.